packages/api-plugin-fulfillment-method-shipping-flat-rate/src/mutations/deleteFlatRateFulfillmentMethod.js

```diff
@@ -0,0 +1,45 @@
+import SimpleSchema from "simpl-schema";
+import ReactionError from "@reactioncommerce/reaction-error";
+
+const inputSchema = new SimpleSchema({
+  methodId: String,
+  shopId: String
+});
+
+/**
+ * @method deleteFlatRateFulfillmentMethodMutation
+ * @summary deletes a flat rate fulfillment method
+ * @param {Object} context - an object containing the per-request state
+ * @param {Object} input - Input (see SimpleSchema)
+ * @returns {Promise<Object>} An object with a `method` property containing the deleted method
+ */
+export default async function deleteFlatRateFulfillmentMethodMutation(context, input) {
```

Comment:
You should not include the word "Mutation" in the name of the function

Reply:
Removed